packages/office-ui-fabric-react/src/utilities/test/TestUtilities.ts

```diff
@@ -0,0 +1,28 @@
+import { ReactWrapper } from 'enzyme';
```

Comment:
Not sure if this is the right place for this, but I imagine these functions will come in handy later